cfgov/jinja2/v1/_includes/molecules/card.html

```diff
@@ -1,17 +1,26 @@
 {# ==========================================================================
 
-   Card
+   card.render( value )
 
    ==========================================================================
 
    Description:
 
-   Creates a card with header and body text.
+   Creates markup for a Card molecule, which displays an icon, some text,
+   and a link.
+
+   value.text:                  Card body text.
+
+   value.link_text:             Card link text.
+
+   value.link_url:              Card link URL.
 
    ========================================================================== #}
 
-{# TODO: This is a prototype and needs to change the text via parameters. #}
-<div class="m-card">
-    <h3 class="h1 m-card_heading">Card Heading</h3>
-    <p class="m-card_body">Card text</p>
+{% macro render( value ) %}
+<div class="o-card-group">
```

Comment:
Should this be `o-card`?

Reply:
You tell me! Maybe it should be `m-card`? Changed in 522dc61eb, but feel free to push to this branch/merge while I'm out for 🦃.